pyccel/parser/semantic.py

```diff
@@ -443,16 +443,19 @@ def get_function(self, name):
 
 
     def get_import(self, name):
-        """."""
+        """
+        Search for a Import object with the given name in the current namespace.
+        Return None if not found.
+        """
 
         imp = None
-
         container = self.namespace
         while container:
 
-            if name in container.imports['imports']:
-                imp =  container.imports['imports'][name]
-                break
+            for key,_ in container.imports['imports'].items():
+                if key == name:
+                    imp = container.imports['imports'][key]
+
```

Comment:
is it necessary to keep loping in the parent containers after finding an import.

Reply:
that's true, I changed my code a lot and I forget that I removed a condition there, I will correct it.